Blazor_Foundation_6_Test_Shared/Documentation/Navigation/TopBar.razor

```diff
@@ -1,6 +1,113 @@
+        </p>
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            A top bar <Code>TopBarWrapper</Code> can have two sections: a left-hand section <Code>TopBarLeft</Code> and a rigth-hand section
+            <Code>TopBarRight</Code>.  On small screens, these sections stack on top of each other.
+        </p>
+        <p>
+            In the below example, our top bar includes a <NavLink href="/Dropdown">dropdown menu</NavLink> , along with a text input field and action button.
+            The dropdown menu inherits the background color of the top bar.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderTitle="Razor" HeaderBGcolor="#5d2b90"
+                     Content="PFRvcEJhcldyYXBwZXI+CiAgICA8VG9wQmFyTGVmdD4KICAgICAgICA8TWVudURyb3Bkb3duPgogICAgICAgICAgICA8bGkgY2xhc3M9Im1lbnUtdGV4dCI+U2l0ZSBUaXRsZTwvbGk+CiAgICAgICAgICAgIDxsaT4KICAgICAgICAgICAgICAgIDxOYXZMaW5rPk9uZTwvTmF2TGluaz4KICAgICAgICAgICAgICAgIDxNZW51VG9wTGV2ZWw+CiAgICAgICAgICAgICAgICAgICAgPGxpPjxOYXZMaW5rIGhyZWY9IiMiPk9uZTwvTmF2TGluaz48L2xpPgogICAgICAgICAgICAgICAgICAgIDxsaT48TmF2TGluayBocmVmPSIjIj5Ud288L05hdkxpbms+PC9saT4KICAgICAgICAgICAgICAgICAgICA8bGk+PE5hdkxpbmsgaHJlZj0iIyI+VGhyZWU8L05hdkxpbms+PC9saT4KICAgICAgICAgICAgICAgIDwvTWVudVRvcExldmVsPgogICAgICAgICAgICA8L2xpPgogICAgICAgICAgICA8bGk+PE5hdkxpbmsgaHJlZj0iIyI+VHdvPC9OYXZMaW5rPjwvbGk+CiAgICAgICAgICAgIDxsaT48TmF2TGluayBocmVmPSIjIj5UaHJlZTwvTmF2TGluaz48L2xpPgogICAgICAgIDwvTWVudURyb3Bkb3duPgogICAgPC9Ub3BCYXJMZWZ0PgogICAgPFRvcEJhclJpZ2h0PgogICAgICAgIDxNZW51RHJvcGRvd24+CiAgICAgICAgICAgIDxNZW51VG9wTGV2ZWw+CiAgICAgICAgICAgICAgICA8bGk+PGlucHV0IHR5cGU9InNlYXJjaCIgcGxhY2Vob2xkZXI9IlNlYXJjaCI+PC9saT4KICAgICAgICAgICAgICAgIDxsaT48QnV0dG9uPlNlYXJjaDwvQnV0dG9uPjwvbGk+CiAgICAgICAgICAgIDwvTWVudVRvcExldmVsPgogICAgICAgIDwvTWVudURyb3Bkb3duPgogICAgPC9Ub3BCYXJSaWdodD4KPC9Ub3BCYXJXcmFwcGVyPg==" />
+        <TopBarWrapper>
+            <TopBarLeft>
+                <OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDropdown>
+                    <li class="menu-text">Site Title</li>
+                    <li>
+                        <NavLink>One</NavLink>
+                        <MenuTopLevel>
+                            <li><NavLink href="#">One</NavLink></li>
+                            <li><NavLink href="#">Two</NavLink></li>
+                            <li><NavLink href="#">Three</NavLink></li>
+                        </MenuTopLevel>
+                    </li>
+                    <li><NavLink href="#">Two</NavLink></li>
+                    <li><NavLink href="#">Three</NavLink></li>
+                </OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDropdown>
+            </TopBarLeft>
+            <TopBarRight>
+                <OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDropdown>
+                    <MenuTopLevel>
+                        <li><input type="search" placeholder="Search"></li>
```

Comment:
@alseny-diallo @Mady-Camara @A-San96 for the example its good since it has no purpose but note: in blazor we use InputText component but you must use EditContext which overkill just for an visual example